style.css

```diff
@@ -1,9 +1,52 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.none {
+  display: none;
+}
+
 main {
   margin: auto;
   width: 400px;
   text-align: center;
 }
 
+.navigation {
+  display: flex;
+  width: 800px;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Please make your project look as similar as possible to the given template. 😄
    - This is your project:
      ![imagen](https://user-images.githubusercontent.com/60789317/143487084-a541976a-53ed-4528-813c-ad585b413086.png)
    - This is the given template:
      ![imagen](https://user-images.githubusercontent.com/60789317/143487140-c5f4c786-119c-4cec-878e-b8ef64e1cc46.png)